cmd/swarm/swarm-smoke/util.go

```diff
@@ -36,83 +38,73 @@ import (
 	"github.com/ethereum/go-ethereum/swarm/api/client"
 	"github.com/ethereum/go-ethereum/swarm/spancontext"
 	opentracing "github.com/opentracing/opentracing-go"
+	"github.com/pborman/uuid"
 	cli "gopkg.in/urfave/cli.v1"
 )
 
 var (
-	commandName = ""
+	commandName       = ""
+	seed        int64 = time.Now().UTC().UnixNano()
 )
 
-func wrapCliCommand(name string, killOnTimeout bool, command func(*cli.Context) error) func(*cli.Context) error {
```

Comment:
Removing timeout handling from `wrapCliCommand`, because we need to handle the timeouts in specific way in all commands.